static/styles/settings.css

```diff
@@ -960,7 +957,7 @@ input[type="checkbox"] {
     font-size: 18px;
     text-align: left;
     margin-top: 0;
-    margin-bottom: 10px;
+    margin-bottom: 20px;
```

Comment:
This CSS is not used anywhere now since you removed the `edit_bot_email` class it is not used anywhere else, which is kinda right since we cannot change the email, so we can actually remove this CSS.

Reply:
Removed the property